components/Menu.tsx

```diff
@@ -138,7 +138,57 @@ export default function Menu({ open, setSidebarOpen }: MenuProps) {
                 <MenuLink href="/intro">Intro</MenuLink>
                 <MenuLink href="/legal">Legal</MenuLink>
                 <MenuLink href="/policy">Policy</MenuLink>
-                <MenuLink href="/reference">Reference</MenuLink>
+                <MenuLink 
+                  href="/reference"
+                  subLinks={
+                    <Fragment>
+                      <MenuSubLink href="/reference#api-versioning">
+                        API Versioning
+                      </MenuSubLink>
+                      <MenuSubLink href="/reference#error-messages">
+                        Error Messages
+                      </MenuSubLink>
+                      <MenuSubLink href="/reference#authentication">
+                        Authentication
+                      </MenuSubLink>
+                      <MenuSubLink href="/reference#encryption">
+                        Encryption
+                      </MenuSubLink>
+                      <MenuSubLink href="/reference#snowflakes">
+                        Snowflakes
+                      </MenuSubLink>
+                      <MenuSubLink href="/reference#id-serialization">
+                        ID Serialization
+                      </MenuSubLink>
+                      <MenuSubLink href="/reference#iso8601-date/time">
+                        ISO8601 Date/Time
+                      </MenuSubLink>
+                      <MenuSubLink href="/reference#nullable-and-optional-resource-fields">
+                        Nullable and Optional Resource Fields
+                      </MenuSubLink>
+                      <MenuSubLink href="/reference#consistency">
+                        Consistency
+                      </MenuSubLink>
+                      <MenuSubLink href="/reference#http-api">
+                        HTTP API
+                      </MenuSubLink>
+                      <MenuSubLink href="/reference#gateway-(websocket)-api">
```

Comment:
```suggestion
                      <MenuSubLink href="/reference#gateway-websocket-api">
```
Believe these get stripped as well